src/Ng/Jobs/Catalog2RegistrationJob.cs

```diff
@@ -57,7 +57,10 @@ public override string GetUsage()
                    + $"-{Arguments.SemVer2StorageAccountName} <azure-acc> "
                    + $"-{Arguments.SemVer2StorageKeyValue} <azure-key> "
                    + $"-{Arguments.SemVer2StorageContainer} <azure-container> "
-                   + $"-{Arguments.SemVer2StoragePath} <path>";
+                   + $"-{Arguments.SemVer2StoragePath} <path>"
+                   + Environment.NewLine
+                   + "Optional Arguments"
+                   + $"-{Arguments.StorageOperationMaxExecutionTimeInSeconds} <azureOperationMaxExecutionTimeInSeconds>";
```

Comment:
Technically, this is a parameter for all of our jobs that use storage accounts now.

We should add a work item to make the `GetUsage` strings generated in a more automated way.

One potential option:
```
public string GetStorageFactoryUsage()
{
    $"-{Arguments.StorageBaseAddress} <storage-base-address> " +
    $"-{Arguments.StorageType} file|azure " +
    ...
}
```